### ISSUE TITLE
Arctos bear in search page has link to Museum of the North

### ISSUE THREAD

**Issue by ewommack** [ArctosDB/arctos#3430]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Describe the bug**
While logged in and working in Arctos I clicked on the Bear Icon and it took me here: http://www.uaf.edu/museum/collections/archaeo/

**To Reproduce**
Go to Arctos search page: https://arctos.database.museum/
click on the Bear Icon up in the upper left corner.

**Expected behavior**
Could that please either link to the Arctos main page or be customizable per collection please? Why am I working in the UWYMV page, but then have a link to go to the UAM Museum of the North public page?

**Desktop (please complete the following information):**
MAC - Chrome

**Priority**
Please assign a priority-label. Unprioritized issues get sent into a black hole of despair.

**Comment by mkoo** [comment]
Are you on test? logged in? I dont see any UAM link on the public arctos.database.museum? screenshot please too?
as both public and logged in user:
Top Arctos bear on public page takes me to arctosdb.org
Bottom bear goes to arctos.database.museum

**Comment by dustymc** [comment]
You must have somehow got to a portal, which can set the behavior of the header.

**Comment by ewommack** [comment]
Nope not on test.

![Arctos bear weird](https://user-images.githubusercontent.com/18562884/107821687-21f27280-6d3a-11eb-9e57-0dc99f763005.jpg)

I've left the page, closed the window, and come back and it's still doing it. 
I had to log out and then log back in for it to stop. 

I was trying to find something in UAM through the portal, but I closed that window and went back to my business. Perhaps that did it? But then why did it keep saying I was in the portal for UAM even when after I left and while I was logged in?
